iroh-resolver/src/hamt/bitfield.rs

```diff
@@ -0,0 +1,168 @@
+use anyhow::{ensure, Result};
```

Comment:
Your frustration was real if you had to do this

Reply:
this is mostly copy and paste from 3 different impls😅